Code/Editor/AzAssetBrowser/AzAssetBrowserWindow.cpp

```diff
@@ -164,6 +168,20 @@ QObject* AzAssetBrowserWindow::createListenerForShowAssetEditorEvent(QObject* pa
     return listener;
 }
 
+void AzAssetBrowserWindow::resizeEvent(QResizeEvent* ev)
+{
+    const QWidget* leftLayout = m_ui->m_leftLayout;
+    const QVBoxLayout* rightLayout = m_ui->m_rightLayout;
+
+    float oldLeftLayoutWidth = aznumeric_cast<float>(leftLayout->geometry().width());
+    float oldWidth = aznumeric_cast<float>(leftLayout->geometry().width() + rightLayout->geometry().width());
+
+    float newWidth = oldLeftLayoutWidth * aznumeric_cast<float>(ev->size().width()) / oldWidth;
```

Comment:
uber nit: These could also be `const`

Reply:
Also the line `float newWidth = oldLeftLayoutWidth * aznumeric_cast<float>(ev->size().width()) / oldWidth;` is a tiny bit confusing, a guide comment might help clarify exactly what's happening